framework/db/ConnectionInterface.php

```diff
@@ -13,6 +13,7 @@
  * ConnectionInterface defines a connection to a database.
  *
  * @author Nelson J Morais <njmorais@gmail.com>
+ * @since 2.0.15
  * @package yii\db
```

Comment:
Yii doesn't use `@package` AFAIK.

Reply:
@sergeymakinen you're right, my bad! done @package tag removed.